src/whiteboard-protocol.js

```diff
@@ -221,17 +218,17 @@ class WhiteboardProtocol {
     return newData;
   }
 
-  ot(data, type) { // other data, 
+  static ot(data, type) { // other data, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5384064)